RoddyCore/src/de/dkfz/roddy/core/Analysis.java

```diff
@@ -514,7 +514,7 @@ private void maybeAbortStartedJobsOfContext(ExecutionContext context) {
         if (Roddy.isStrictModeEnabled() && context.getFeatureToggleStatus(AvailableFeatureToggles.RollbackOnWorkflowError)) {
             try {
                 logger.severe("A workflow error occurred, try to rollback / abort submitted jobs.");
-                Roddy.getJobManager().queryJobAbortion(context.jobsForProcess);
+                Roddy.getJobManager().killJobs(context.jobsForProcess);
```

Comment:
In addition to my comment in the current BatchEuphoria PR. killJobs also does not guarantee, that the jobs really get killed. E.g. the command might return without an error, but the jobs stay in the queue.

Reply:
I mentioned it because of the method rename :D. The issue itself is not an easy task and definitely out of scope.